ingest-app/src/migrations/008_add_name_column_bulk_update_status.clj

```diff
@@ -0,0 +1,22 @@
+(ns migrations.008-add-name-column-bulk-update-status
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 8."
+  []
+  (println "migrations.008-add-name-column-bulk-update-status up...")
+  (j/db-do-commands (config/db)
+                    "ALTER TABLE bulk_update_task_status ADD
+                     name VARCHAR(255) DEFAULT NULL")
+  (j/db-do-commands (config/db)
+                    "UPDATE bulk_update_task_status SET name = task_id
+                     WHERE name IS NULL"))
```

Comment:
I think this comment was missed when changing the migration. We should not allow inserting NULLs after this migration has run. So we need to add a 3rd step after updating everything that had a NULL value to no longer allow NULLs:

```
(j/db-do-commands (config/db)
                  "ALTER TABLE CMR_INGEST.bulk_update_task_status
                   MODIFY name VARCHAR(255) NOT NULL")
```